adc.c

```diff
@@ -0,0 +1,41 @@
+
+#include "system.h"
+#include "adc.h"
+#include "nuts_bolts.h"
+
+#define ADMUX_SELECTION_MASK    0x7
+#define MUX5_MASK               0x8
+
+uint16_t adc_read_channel(uint8_t channel) {
+  // Set ADC reference
+  ADMUX = (1 << REFS0);
+
+  // Enable ADC, Prescaler: 128x
+  ADCSRA = (1 << ADEN) | (1 << ADPS2) | (1 << ADPS1) | (1 << ADPS0);
+
+  // Select Channel
+  ADMUX |= (channel & ADMUX_SELECTION_MASK);
+
+  // Select Channel - ADCSRB, MUX5
+  if (channel & MUX5_MASK) {
```

Comment:
right.  Is it true that ADC 8-15 all set that mux bit high and then use the lower bits as the "address".

I know that ADC15 is basically a read of adc 7 with that extra mux bit set.  Is the same tru for 8-14?